### PR TITLE
Added EE TerminateThread test.

### PR DESCRIPTION
**Pull request by jpd002**
Added test cases for TerminateThread function. 

I've also added a "getThreadPriority" helper function which I'm using to get the test thread priority instead of using a `#define` that's duplicated everywhere. I'm going to make other kernel tests use that function in another PR.

### CODE REVIEW

File: common/common-ee.h [hunk 1]
```diff
@@ -17,4 +17,6 @@
 void schedf(const char *format, ...);
 void flushschedf();
 
+int getThreadPriority(int);
```
Comment (unknownbrackets) [inline common/common-ee.h]: I'd prefer this had `int threadId` to ensure it's clear (in IDEs or by browsing) what the argument is used for.

-[Unknown]

File: common/common-ee.cpp [hunk 2]
```diff
@@ -22,6 +25,17 @@ void flushschedf() {
 	schedfBufferPos = 0;
 }
 
+int getThreadPriority(int threadId) {
+	ee_thread_status_t threadStat;
+	memset(&threadStat, 0, sizeof(ee_thread_status_t));
+	int result = ReferThreadStatus(threadId, &threadStat);
+	if(result >= 0) {
```
Comment (unknownbrackets) [inline common/common-ee.cpp]: Nit: space after `if` in common code, please.

-[Unknown]

**Comment by unknownbrackets** [comment]
Hmm, it might make sense to add a log line after starting the thread a second time just to clarify that it can be reused (before terminate after suspend)... though this test really already shows that.  Seems pretty good.

-[Unknown]

**Comment by jpd002** [comment]
Fixed code style issues and clarified the thing about starting a terminated thread.

**Pull request merged**